tests/firehdf_test.py

```diff
@@ -0,0 +1,54 @@
+import gc
+
+import numpy as np
+import pytest
+
+import pynbody
+import pynbody.test_utils
+
+# PENDING: find/generate smaller test data, upload to zenodo
+"""
+@pytest.fixture(scope='module', autouse=True)
+def get_data():
+    pynbody.test_utils.ensure_test_data_available("pkdgrav3")
+
+"""
+
+@pytest.fixture
+def snap():
+    f = pynbody.load('testdata/FIRE/m11i_res7100/output/snapshot_600.hdf5')
+    yield f
+    del f
+    gc.collect()
+
+
+@pytest.fixture
+def multi_snap():
+    f = pynbody.load('testdata/FIRE/m10q_res30/output/snapdir_600/snapshot_600')
```

Comment:
Creating a small multi-file sample is harder due to the distributed information across different files. But given that the multi-file handling is not really different in Gizmo/FIRE vs baseline GadgetHDF, I'm not sure we have to test the multi-file case unless there is a particular reason to do so.